XP_Pen_Deco01/index.md

```diff
@@ -0,0 +1,20 @@
+---
+VID: "28bd"
+PID: "0042"
+vendor: Ugee
+product: Deco_01
+image: XP_Pen_Deco
+report_rate: 233
+working_area:
+    width: 10
+    height: 6.25
+    resolution: 5080
+pen:
+    pressure_levels: 8192
+frame_controls: 8 buttons
+sold_as:
+    - XP-Pen Deco 01
+    - XP-Pen Deco 01 Digital Graphics Drawing Tablet
```

Comment:
These sound very similar. Let's not repeat ourselves and not create unnecessary noise in the data and just name it the official way manufacturer names them on their website, that is "XP-Pen Deco 01". All these extra words are just keywords for search engines sellers add.

Could you please also cleanup the entries you submitted earlier, regarding this?